src/sentry/middleware/health.py

```diff
@@ -19,9 +21,16 @@ def process_request(self, request):
 
         from sentry.status_checks import check_all
         from sentry.utils import json
-        problems, checks = check_all()
 
+        results = check_all()
```

Comment:
I don't understand this comment. Are you talking about some sort of severity level threshold filter to avoid displaying warnings by default?